File: Estrutura_Condicional/Python/Ex2702.py
disponiveis = list(map(int, input().split()))
requisitadas = list(map(int, input().split()))

nao_atendidos = 0
for i in range(3):
    nao_atendidos += max(0, requisitadas[i] - disponiveis[i])

print(nao_atendidos)

File: Estrutura_Condicional/JavaScript/Ex2702.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

var disponiveis = prompt().split(" ").map(Number);
var requisitadas = prompt().split(" ").map(Number);

var nao_atendidos = 0;
for (var i = 0; i < 3; i++) {
    nao_atendidos += Math.max(0, requisitadas[i] - disponiveis[i]);
}

console.log(nao_atendidos);